src/api/user/src/services/firestore.js

```diff
@@ -0,0 +1,32 @@
+const admin = require('firebase-admin');
+const { logger } = require('@senecacdot/satellite');
+const path = require('path');
+require('dotenv').config({ path: path.resolve('../env.development') });
```

Comment:
We can't do this, since we need to use whichever ENV variables are defined.  You should just use process.env.* and not pull it in.  In tests we can, because we control the environment, but we don't for staging/production.